README.md

```diff
@@ -65,6 +65,79 @@ const { GenericContainer } = require("testcontainers");
 })();
 ```
 
+Passing arguments to your own Docker image:
```

Comment:
I'm not sure why these are showing as additions?

Reply:
Not sure either...